solidity/test/data/moving-funds.ts

```diff
@@ -0,0 +1,587 @@
+    "0xaf7a841e055fc19bf31acf4cbed5ef548a2cc453",
+  ],
+
+  mainUtxo: {
+    txHash:
+      "0x3ca370f492449e2b413c27fe18b1a139f602d049fc0903595c41441f5a35eaed",
+    txOutputIndex: 0,
+    txOutputValue: 1279418,
+  },
+
+  // https://live.blockcypher.com/btc-testnet/tx/5b362273b9471f62c35b2ef6940c3c13bd2b773626ab5de63efe4e895d56f98c
+  movingFundsTx: {
+    hash: "0x8cf9565d894efe3ee65dab2636772bbd133c0c94f62e5bc3621f47b97322365b",
+    version: "0x01000000",
+    inputVector:
+      "0x013ca370f492449e2b413c27fe18b1a139f602d049fc0903595c41441f5a35eaed" +
+      "0000000000ffffffff",
+    outputVector:
+      "0x032f760600000000001976a9142cd680318747b720d67bf4246eb7403b476adb34" +
+      "88ac30760600000000001600148900de8fc6e4cd1db4c7ab0759d28503b4cb0ab133" +
+      "760600000000001976a914af7a841e055fc19bf31acf4cbed5ef548a2cc45388ac",
+    locktime: "0x00000000",
+  },
+
+  movingFundsProof: {
+    merkleProof:
+      "0x24c50204f1d2c838e7872dd7f5c3a38bac573cce654620dfec76c6a05e9eb3139d" +
+      "b0514e107025c8c055c8a10e83826c244820a32821a3d010880c1b3ab6bcf620fb48" +
+      "f188126e2628fb2d3ead8b7bd458c0bc9d2b6d15368831538114901f3bfd9ad0532a" +
+      "b93c6f36c1755d3315fbdd146753b2332363cf75b931ddb824e9c3e21676a253e14f" +
+      "be05ca546420c7b2e535666129660fb462adde0075114d5e00",
+    txIndexInBlock: 2,
+    bitcoinHeaders:
+      "0x00000020fea86838d788d0c0e2f4c32272ba5dd599c4bd04baa4ff759900000000" +
+      "000000b04fb0eaa2f1e39b53c26afdfa726289940864e3a42e06103fc7236aa67b15" +
+      "f3d50c50621ec8001a73dc826b04000020b18b27f95a438fd3d1bd571fd5ed10876b" +
+      "678f7f29cbb9a3ba000000000000009f9da0db22cc9d10d085502e74e6025e295bef" +
+      "3abc1441f7f0986ddadfd97776ce0d50621ec8001a7b784f6200000020e17a6b1a17" +
+      "34108e303b47a27ac7ff80a9ce9facc795e5bf8600000000000000e6da2811b1b28d" +
+      "b55ac59062c3651798356c5186b92664628eae8bc14f634ec27b0e50621ec8001a2e" +
+      "4075860000c020e7021f51166a77246660d627322dc6887cf0a9491f07f3d9430000" +
+      "00000000005c5437498047dcc617029442e5493460987ce6a1d186c2562412eae524" +
+      "42aa79b20e50621ec8001a02d802c804000020007219023a7eea3691d8091f3e67a2" +
+      "c17bbe15da9ae464663c000000000000007e1b1140ada8cb3f240f011243c8da5e4d" +
+      "abcd753d22d13c86d7c240413165f6160f50621ec8001a57daeccc0000a020a8597b" +
+      "1c1720560be7cc0f3707499f11e05df742c74cb5c99f00000000000000e17e1fc4b6" +
+      "70a70c0c67b1ccb183f0e8bf4c3321039d49675aa6886bd2ec691c4e0f50621ec800" +
+      "1a59b3b33c000000205058b8dcbbfdfb9e00b30e3f374ec17e1789fc4f3f678f27bf" +
+      "000000000000001b329dd98f5f1d79f36d0b74fa3da4a9e82609a424287f6e1de301" +
+      "485278b0ce4f1150621ec8001a4b24dd9e",
+  },
+
+  chainDifficulty: 21461933,
+}
+
+/**
+ * `SingleTargetWalletButP2SH` test data represents a moving funds with the
+ * following properties:
+ * - 1 main UTXO input
+ * - 1 P2SH output that matches the target wallet from the commitment
+ * - 6+ on-chain confirmations of the sweep transaction
+ */
```

Comment:
We could note it is not a valid moving funds transaction and that it is going to be rejected by the bridge.